00-HTML-CSS-basics/CSS/Layout 2.5.2/css/style.css

```diff
@@ -0,0 +1,133 @@
+
+/*font styles*/
+/*----------*/
+
+@font-face {
+  font-family: 'MyFont';
+  src:url('../fonts/open-sans/OpenSans-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+  font-size: 14px;
+ 
+}
+
+/*element selectors*/
+/*----------------*/
+
+/*applying box-sizing property in order to stablish same-sized boxes*/
+* {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+html {
+  font-family: MyFont, Ubuntu, Arial, TimesNewRoman, sans-serif;
+}
+
+body{
+  background-color: white;
+
+}
+
+header{
+  top: 0;
+}
+
+/*Class elements*/
+/*--------------*/
+
+.header {
+  padding-top: 1rem;
+  width: 100%;
+  font-size: 2rem;
+  text-align: left;
+}
+
+.text-alignment-center{
+   text-align: center;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 48 rem;
+  width: 90%;
+  opacity: 0.8;
+}
+
+
+.clearfix:before,
+.clearfix:after {
+  content: " ";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+.clearfix {
+  *zoom: 1;
+}
+
+
+.odd-section {
```

Comment:
This is the structure defined in http://adamkaplan.me/grid/ (step 7)

the .clearfix is defined in order to have compatibility with IE 6/7 browsers